### PR TITLE
Adding an issue template for the designer

### PR DESCRIPTION
**Pull request by rahulamlekar**
Adding an issue template for the designer. Trying the newer issue template creation flow

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5976)

### CODE REVIEW

**Comment by rahulamlekar** [comment]
GitHub has a new [issue template creation flow](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) which they recommend we use. This creates the template md files for us. The new lines and label reordering in the PR is caused by the new flow and should only be a one-time thing if we use this going forward.

File: .github/ISSUE_TEMPLATE/designer-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+# Host Application
+
+Please provide the name of the application that is rendering the card (ex: Bot Framework, Outlook, Teams, etc)
+
+# Target Version
+
+Please provide the Adaptive Cards SDK version (ex: 1.3, 1.4)
```
Comment (golddove) [inline .github/ISSUE_TEMPLATE/designer-bug-report.md:16]: Do we want SDK version or card schema version?

Comment (matthidinger) [reply .github/ISSUE_TEMPLATE/designer-bug-report.md:16]: Hmm good call, I don't think we need Target version or Host app in this bug type

Comment (rahulamlekar) [reply .github/ISSUE_TEMPLATE/designer-bug-report.md]: Good point, removed.

**Review by golddove — APPROVED** [review]

**Pull request merged**